pygotham/frontend/talks.py

```diff
@@ -60,7 +65,11 @@ def index():
     '/new/',
     defaults={'pk': None},
     endpoint='submit',
-    methods=('GET', 'POST'))
+    methods=('GET', 'POST'),
+    navbar_kwargs={
+        'path': ('Speaking', 'Submit a talk'),
+        'when': lambda: g.current_event.is_call_for_proposals_active,
```

Comment:
Unrelated, but we should probably rename this to `call_for_proposals_is_active` to be consistent with the others.